### PR TITLE
makes the math storyteller does actually useful

### PR DESCRIPTION
**Pull request by vode-code**
## About The Pull Request
This PR changes the re-occurrence math to a spot that Nestor appears to have wanted it.

## Why It's Good For The Game
this change might be good for storyteller.

## Testing
ran a local for a while

## Changelog
:cl: Chickenish
balance: storyteller works differently now.
/:cl:

### CODE REVIEW

**Comment by vode-code** [comment]
SPCR Forklift Bug

File: code/game/gamemodes/storyteller.dm [hunk 1]
```diff
@@ -153,6 +154,9 @@ GLOBAL_VAR_INIT(chaos_level, 1) //Works as global multiplier for all storyteller
 
 	last_tick = world.time
 	next_tick = last_tick + tick_interval
+	if(REALTIMEOFDAY > last_chaos_hour + 1 HOUR)
+		GLOB.chaos_level += 1
+		last_chaos_hour += 1 HOUR
```
Comment (MLGTASTICa) [inline code/game/gamemodes/storyteller.dm:159]: Stealth change and reimplementation of the  previously reverted PR..

Comment (vode-code) [reply code/game/gamemodes/storyteller.dm:159]: ... I merged your revert, that wasn't supposed to be a thing.

Comment (vode-code) [reply code/game/gamemodes/storyteller.dm:159]: why didn't your revert remove it?

Comment (vode-code) [reply code/game/gamemodes/storyteller.dm:159]: it "removed the feature", right?
why did you let the var stay?

Comment (vode-code) [reply code/game/gamemodes/storyteller.dm:159]: wait no
this was just merging before your PR was merged, I think.
will fix.

**Pull request merged**